src/main/scala/zio/redis/package.scala

```diff
@@ -0,0 +1,159 @@
+package zio
+
+package object redis {
+
+  import Command.Input._
+  import Command.Output._
+
+  object connection {
```

Comment:
On second thought, let's make these traits, which are extended by these objects. That way we can have a `all` package, so users can do, `import redis.all._` or something.

Reply:
Makes sense.